src/main/java/org/demo/conf/cxbox/extension/drilldown/CxboxUrlParametersFilterService.java

```diff
@@ -0,0 +1,39 @@
+package org.demo.conf.cxbox.extension.drilldown;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.cxbox.api.data.dto.DataResponseDTO;
+import org.cxbox.constgen.DtoField;
+import org.cxbox.core.service.drilldown.filter.PlatformUrlParametersFilterService;
+import org.cxbox.dictionary.Dictionary;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.stereotype.Service;
+
+/**
+ * Example of create bean PlatformUrlParametersFilterService
+ */
+@Service
+@Slf4j
+public class CxboxUrlParametersFilterService extends PlatformUrlParametersFilterService {
```

Comment:
move this doc java doc and code-samples. demo should stay as compact as possible.